src/components/Figure/index.js

```diff
@@ -102,6 +117,41 @@ export default function Figure() {
     canvas.addEventListener("mouseup", onMouseUp, false);
     canvas.addEventListener("mouseout", onMouseUp, false);
     canvas.addEventListener("mousemove", onMouseMove, false);
+
+    socketRef.current.on("drawing", (data) => {
+      if (data === "triangle") {
+        objects.current.push({
+          x: 250,
+          y: 300,
+          width: 100,
+          height: 50,
+          color: "black",
+          type: "Triangle",
+        });
+      } else if (data === "circle") {
+        objects.current.push({
+          x: 320,
+          y: 250,
+          width: 50,
+          height: 50,
+          color: "black",
+          type: "Circle",
+        });
+      } else if (data === "square") {
+        objects.current.push({
+          x: 320,
+          y: 250,
+          width: 50,
+          height: 50,
+          color: "black",
+          type: "Rect",
```

Comment:
이 부분도 다른 도형들처럼 수정하는게 통일성 있고 좋을 것 같습니다.
```suggestion
          type: "Square",
```